File: src/main/java/org/jukeboxmc/world/BlockUpdateNormal.java
package org.jukeboxmc.world;

import lombok.AllArgsConstructor;
import lombok.Data;
import org.jukeboxmc.block.Block;
import org.jukeboxmc.block.direction.BlockFace;

/**
 * @author LucGamesYT
 * @version 1.0
 */
@Data
@AllArgsConstructor
public class BlockUpdateNormal {

    private Block block;
    private BlockFace blockFace;

}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/block/BlockUpdateNormal.kt
package org.jukeboxmc.server.block

import org.jukeboxmc.api.block.Block
import org.jukeboxmc.api.block.data.BlockFace

class BlockUpdateNormal(
    private val block: Block,
    private val blockFace: BlockFace
) {

    fun getBlock(): Block {
        return this.block
    }

    fun getBlockFace(): BlockFace {
        return this.blockFace
    }

}